app/views/docs/email-and-sms-templates.phtml

```diff
@@ -0,0 +1,145 @@
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 200px">Variable</td>
+        <td>Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>{{project}}</code></td>
+        <td>The project name.</td>
+    </tr>
+    <tr>
+        <td><code>{{team}}</code></td>
+        <td>Thee project team's name.</td>
+    </tr>
+    <tr>
+        <td><code>{{user}}</code></td>
+        <td>The name of the user receiving the email.</td>
+    </tr>
+    <tr>
+        <td><code>{{redirect}}</code></td>
+        <td>The URL for the user to complete the email template's action.</td>
+    </tr>
+    </tbody>
+</table>
+
+<h3><a href="#email-template-examples" id="email-template-components">Email Template Components</a></h3>
+[TODO: @dlohani once we're ready, let's do examples for each type of template (maybe also what it would look like rendered)]
+
+<h2><a href="#sms-templates" id="sms-templates">SMS Templates</a></h2>
+<p>You can customize the SMS emplates used for phone verification and login.</p>
+<h3><a href="#sms-template-components" id="sms-template-components">SMS Template Components</a></h3>
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 200px">Component</td>
+        <td>Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>Message</td>
+        <td>The body of the SMS message. You can find the variables available in the <a href="#sms-template-syntax">SMS Template Syntax</a> section.</td>
+    </tr>
+    </tbody>
+</table>
+
+<h3><a href="#sms-template-syntax" id="sms-template-syntax">SMS Template Syntax</a></h3>
+<p>Variables can be used in SMS templates to dynamically format unique messages for each reader.</p>
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 200px">Variable</td>
+        <td>Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>{{project}}</code></td>
+        <td>The project name.</td>
+    </tr>
+    <tr>
+        <td><code>{{team}}</code></td>
+        <td>Thee project team's name.</td>
+    </tr>
+    <tr>
+        <td><code>{{user}}</code></td>
+        <td>The name of the uer receiving the email.</td>
+    </tr>
+    <tr>
+        <td><code>{{passcode}}</code></td>
+        <td>The secret code sent to users for phone verification or authentication.</td>
+    </tr>
+    </tbody>
+</table>
+
+<h3><a href="#sms-template-examples" id="sms-template-components">SMS Template Components</a></h3>
+[TODO: @dlohani once we're ready, let's do examples for each type of template (maybe also what it would look like rendered)]
+
+<h2>Localization</h2>
+[TODO: @dlohani short description of how it's decided which language's template is sent]
+[TODO: @dlohani Maybe example of a template in two languages? (maybe also what it would look like rendered)]
```

Comment:
what do you mean by this?

Reply:
So we would just describe how users would set their templates, then show the same message for an email template in two languages + screenshots.

This just fully demonstrates how localization works.

Reply:
by using `client.setLocale` to set the locale header.